apps/ledger-live-mobile/src/newArch/features/AssetSelection/screens/SelectNetwork/index.tsx

```diff
@@ -182,29 +53,37 @@ export default function SelectNetwork({
   return (
     <>
       <TrackScreen category="Deposit" name="Choose a network" />
-      <Flex px={6} py={2}>
-        <Text variant="h4" fontWeight="semiBold" testID="receive-header-step2-title" mb={2}>
-          {t("transfer.receive.selectNetwork.title")}
-        </Text>
-        <Text
-          variant="bodyLineHeight"
-          fontWeight="medium"
-          color="neutral.c80"
-          testID="receive-header-step2-subtitle"
-        >
-          {t("transfer.receive.selectNetwork.subtitle")}
+      <Flex px={6} py={2} testID="select-network-view-area">
+        <Text variant="h4" fontWeight="semiBold" testID={titleTestId} mb={2}>
+          {titleText}
         </Text>
+        {subtitleText && (
+          <Text
+            variant="bodyLineHeight"
+            fontWeight="medium"
+            color="neutral.c80"
+            testID={subTitleTestId}
+          >
+            {subtitleText}
+          </Text>
+        )}
       </Flex>
       <Flex ml={16} mr={16} flex={1}>
-        <FlatList
-          testID="receive-header-step2-networks"
-          contentContainerStyle={styles.list}
-          data={sortedCryptoCurrenciesWithAccounts}
-          renderItem={renderItem}
-          keyExtractor={keyExtractor}
-          showsVerticalScrollIndicator={false}
-          keyboardDismissMode="on-drag"
-        />
+        {["success", "error"].includes(providersLoadingStatus) ? (
```

Comment:
why not use a type or enum here?

Reply:
`providersLoadingStatus` is already a literal string type with 4 variants

Reply:
replaced the string by the corresponding enums.